js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
Something like this
```js
    AddressBookService.getAll().then(function(addressBooks) {
        if(ctrl.addressBooks.length === 0) {
            return AddressBookService.create(t('contacts', 'Contacts')).then(function(addressBook) {
                return [addressBook];
            });
        }
        return addressBooks;
    }).then(function(addressBooks) {
        ctrl.addressBooks = addressBooks;
        ctrl.loading = false;
        $scope.$apply();
    });
```

It more or less does the same though, it's just a different coding style 😉